libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -474,7 +474,7 @@ - (SFUserAccount*)createUserAccount {
     SFUserAccount *newAcct = [[SFUserAccount alloc] initWithIdentifier:[self uniqueUserAccountIdentifier]];
     SFOAuthCredentials *creds = newAcct.credentials;
     creds.accessToken = nil;
-    creds.domain = self.loginHost;
+    creds.domain = [self.loginHost containsString:@"?display="]?self.loginHost:[self.loginHost stringByAppendingString:@"?display=page"];
```

Comment:
Why are we doing this? We've always used `?display=touch` on mobile and in recent times I thought the display param was being deprecated. Also, I don't think domain should have a `display` param. Besides, how was it working on `UIWebView` and only failing on `WKWebView`?